rakelib/artifacts.rake

```diff
@@ -43,7 +43,17 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
-  task "prepare" => ["bootstrap", "plugin:install-default"]
+  task "use-defaults-gemfile" do
+    FileUtils.cp("Gemfile.defaults", "Gemfile")
```

Comment:
Alternately, if it's only part of package building, we don't have to overwrite anything, we can make the "copy" happen during packaging.